modules/atomic-widgets/widgets/atomic-heading.php

```diff
@@ -44,6 +54,27 @@ protected function render() {
 		);
 	}
 
+	public function get_link_template( array $settings ): string {
```

Comment:
Seems like we are duplicating a lot of the logic here. We could just conditionally render an `<a>` tag inside the original render method? 🤔